dags/adm_export.py

```diff
@@ -0,0 +1,73 @@
+import datetime
+
+from airflow import DAG
+from airflow.hooks.base_hook import BaseHook
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+
+from operators.gcp_container_operator import GKEPodOperator
+from operators.task_sensor import ExternalTaskCompletedSensor
+
+
+DOCS = """\
+Weekly data exports of contextual services data aggregates to adMarketplace.
+This is a complementary approach to the near real-time sharing that is implemented
+in gcp-ingestion.
+
+For more context, see https://bugzilla.mozilla.org/show_bug.cgi?id=1729524
+"""
+
+default_args = {
+    "owner": "jklukas@mozilla.com",
+    "start_date": datetime.datetime(2019, 7, 25),
+    "email": ["telemetry-alerts@mozilla.com", "jklukas@mozilla.com"],
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "depends_on_past": False,
+    # If a task fails, retry it once after waiting at least 5 minutes
+    "retries": 1,
+    "retry_delay": datetime.timedelta(minutes=5),
+}
+
+dag_name = "adm_export"
+
+with DAG(
+    dag_name, schedule_interval="0 5 * * *", doc_md=DOCS, default_args=default_args
+) as dag:
+
+    conn = BaseHook.get_connection('adm_sftp')
+
+    adm_weekly_aggregates_to_sftp = GKEPodOperator(
+        task_id="adm_weekly_aggregates_to_sftp",
+        name="adm_weekly_aggregates_to_sftp",
+        # See https://github.com/mozilla/docker-etl/pull/28
+        image="gcr.io/moz-fx-data-airflow-prod-88e0/bq2sftp_docker_etl:latest",
+        project_id=GoogleCloudBaseHook(gcp_conn_id=gcp_gke_conn_id).project_id,
```

Comment:
i dont see gcp_gke_conn_id defined?

Reply:
if this needs to be revised, can you just hardcode it to 'moz-fx-data-airflow-gke-prod', since 2.0 will need it this way

Reply:
You're right. I copy/pasted this. I had set auto-merge and I was expecting this to not merge if the test-environment task failed. Apologies for the churn. I'm working on a follow-up immediately.